HACKING.md

```diff
@@ -299,3 +310,38 @@ http://simple-httpd-fedora-coreos.$CLUSTER_URL
 ```
 
 (Or simply check the output of `oc get route simple-httpd`).
+
+
+
```

Comment:
Minor: lots of extra blank spaces here

Reply:
Yeah. didn't know if the extra spaces would help delineate the next heading as a new **major** section or not. Can you view this in rich format and let me know the best way to do that?

Reply:
either way I deleted the extra space.

Reply:
You can see it here: https://github.com/coreos/fedora-coreos-pipeline/blob/b2c9ec407ecb56b274c6b9377447c69c6e482080/HACKING.md.

AFAIK, markdown doesn't care about the number of spaces. Just using the appropriate number of `#` signs should be good. :+1: